CHANGELOG.md

```diff
@@ -111,6 +111,7 @@ It includes many new rules, including all new techniques introduced in MITRE ATT
 - linter: summarize results at the end #571 @williballenthin
 - meta: added `library_functions` field, `feature_counts.functions` does not include library functions any more #562 @mr-tz
 - linter: check for `or` with always true child statement, e.g. `optional`, colors #348 @mr-tz
+- json: breaking change: record all matching strings for regex #159 @williballenthin
```

Comment:
i think we should separate out the breaking changes and highlight them for our users

Reply:
@Ana06

Reply:
A `Breaking Changes` section inside the `Changes` section ? That's a good idea! We should also add it to https://github.com/fireeye/capa/blob/master/doc/release.md

Should we create this section now or when we release? 🤔

Reply:
see #597 I'd say add a `### Breaking Changes` section now.